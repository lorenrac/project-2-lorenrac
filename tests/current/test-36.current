ERROR (line 3): Variable 'my_var' already declared in this scope
