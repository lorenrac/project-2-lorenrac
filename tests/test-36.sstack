// ERROR: Redeclaring a variable in the same scope.
VAR my_var = "good"
VAR my_var = "bad"
PRINT my_var
