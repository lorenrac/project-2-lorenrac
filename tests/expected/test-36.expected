ERROR (line 3): Redeclaration of variable 'my_var' (originally defined on line 2)
